docs(example): add an example of using remix-auth-email-link (#1365)
remix-run/remix: examples/remix-auth-email-link/README.md

# Remix Auth Email Link

Passwordless authentication by sending a magic link to your email. This example is using [remix-auth](https://github.com/sergiodxa/remix-auth) and [remix-auth-email-link](https://github.com/pbteja1998/remix-auth-email-link) packages.

## Preview

Open this example on [CodeSandbox](https://codesandbox.com):

> [![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-email-link)

Reviewer: Several people are making examples of strategies with remix-auth and we decided we want to group them in a `examples/remix-auth` directory. Could you move this example into a sub directory (`examples/remix-auth/email-link`)?

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/email-link)
```

> [![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-email-link)

Reviewer 2: ```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-email-link)
```

## Example

> This is using `remix-auth` and `remix-auth-email-link` packages. Instead of actually sending the magic link to the email, in this example I am just logging it on to the console. You can take care of that implementation yourself.

Reviewer: ```suggestion
This is using `remix-auth` and `remix-auth-email-link` packages. Instead of actually sending the magic link to the email, in this example I am just logging it on to the console. You can then copy/paste that into the URL. For sending the email you'll likely want to use an email service like SendGrid or Mailgun.
```

> 

Reviewer: Could you list and link to the relevant files with a short description of what each does?
